ui/src/Constant/constant.js

```diff
@@ -11,3 +11,20 @@ export const TABLE = {
     "ENABLED":"Enabled",
 }
 
+
+
+export const NAV = {
+    "Home" : "/",
+    "Create Users" : "/users",
+    "Flags" : "/flags",
+    "SDK" : "/sdk",
+    "Docs" : "/docs",
+}
+
+export const PATH = {
+    "HOME" : "/",
+    "CREATE_USERS" : "/users",
+    "FLAGS" : "/flags",
+    "SDK" : "/sdk",
+    "DOCS" : "/docs",
+}
```

Comment:
I think we are violating DRY principle here. 
How about?
```suggestion
export const NAV = {
    "Home" : PATH.HOME,
    "Create Users" : PATH.CREATE_USERS,
    "Flags" : PATH.FLAGS,
    "SDK" : PATH.SDK,
    "Docs" : PATH.DOCS,
}

export const PATH = {
    "HOME" : "/",
    "CREATE_USERS" : "/users",
    "FLAGS" : "/flags",
    "SDK" : "/sdk",
    "DOCS" : "/docs",
}
```

Reply:
I am thinking to create an array object. An object will have a name and URL. Something like this:

and loop over to create the navigation and URL.

One down point I see in this is scaling to different languages could be not easy. WYT?

```
export const NAV = [
    {
        name: 'Home',
        path: '/'
    },
    {
        name: "Create Users",
        path:  "/users"

    },
    {
        name: "Flags",
        path:  "/flags"
    },
    {
        name: "SDK",
        path:  "/SDK"
    },
    {
        name: "Docs",
        path:  "/docs"
    }
]
```

Reply:
Sounds good.